### ISSUE TITLE
Cannot find module 'virtual:vite-plugin-sentry/sentry-release'

### ISSUE THREAD

**Issue by callmesoul** [ikenfin/vite-plugin-sentry#19]
Cannot find module 'virtual:vite-plugin-sentry/sentry-release'

**Comment by callmesoul** [comment]
vite by 2.9.12

**Comment by callmesoul** [comment]
Is there a use case for vite-plugin-sentry？

**Comment by ikenfin** [comment]
Just checked up it with vite 2.9.12 and vue3 sample app and seems it works. 
Please make sure you are using latest vite-plugin-sentry version (at now 1.0.18) - this virtual import feature was added in 1.0.17.

Also here is my test vue setup example:

```js
import { createApp } from 'vue'
import App from './App.vue'

import * as Sentry from "@sentry/vue"
import { BrowserTracing } from "@sentry/tracing";

import 'virtual:vite-plugin-sentry/sentry-release'

const app = createApp(App)

Sentry.init({
  app,
  dsn: "https://<SENTRY_DSN_URL_HERE>",
  integrations: [
    new BrowserTracing({
      tracingOrigins: [/.*/]
    }),
  ],
  tracesSampleRate: 1.0,
  release: import.meta.env.SENTRY_RELEASE.id,
  environment: process.env.NODE_ENV
})

app.mount('#app')

```

**Comment by ikenfin** [comment]
@callmesoul Hi! Seems that our ssr fixes affects to dev mode, i will fix it asap 👍

**Comment by ikenfin** [comment]
@callmesoul just published update to npm v1.0.19, please check if it fixed your problem
